app/views/feeds/_tabs.html.erb

```diff
@@ -12,12 +12,17 @@
     <div class="fixed-bottom col-sm-12 col-md-3 col-xl-2">
       <div class="btn-group dropup d-flex sticky-md-bottom m-4 shadow">
         <%= link_to "Add a Feed", new_podcast_feed_path(podcast), class: "btn btn-success flex-grow-1" %>
-        <% if @feeds.none?(&:apple?) %>
+        <% if @feeds.none?(&:integration_type) %>
```

Comment:
This will need to change - if you have an apple feed, you can't add a megaphone one etc.

But for now, I think we should just hide the Megaphone button until we're ready to go live.

Reply:
hidden!